src/controllers/genresControllers.ts

```diff
@@ -0,0 +1,48 @@
+import { Request, Response } from 'express';
+
+import { Genre, GenreObj } from '../models';
+import { errorHandler } from '../middleware/errorHandlers';
+
+export const getAllGenres = async (req: Request, res: Response) => {
+  const genres: GenreObj[] = await Genre.find();
+  if (!genres.length) {
+    return res.sendStatus(204);
```

Comment:
In such cases, it is more appropriate to send an empty array with a status code of 200
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204